features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/source/src/app/ProtectedApp.js

```diff
@@ -86,10 +86,20 @@ export default class ProtectedApp extends Component {
     componentDidMount() {
         ConfigManager.getConfigs().environments.then(response => {
             this.environments = response.data.environments;
-            this.handleEnvironmentQueryParam();
+            //this.handleEnvironmentQueryParam(); todo: do we really need to handle environment query params here ?
```

Comment:
Yes multi-environment feature is not available in APIM 3.x So we could remove this `handleEnvironmentQueryParam` from here, But make sure it won't brake any other functionalities.

Reply:
Don't keep the commented code.

Reply:
resolved by #8332